django/cantusdb_project/main_app/models/chant.py

```diff
@@ -148,3 +149,49 @@ def get_next_folio(folio):
             next_chant = None
 
         return next_chant
+
+    def get_syllabized_text_and_melody(self, text_only=False):
+        """Returns the syllabized text with the melody of a chant. Syllabized text splits the words from the
+         full text into its syllable components based on the volpiano protocols
+         (ex: Ocundare plebs fidelis cuius pater => I-O-cun-da-re plebs fi-de-lis cu-ius pa-ter)
```

Comment:
Oops, I think the `I-` at the beginning of your example is a typo?

Reply:
the word actually is Iocundare, and *should* be syllabized Io-cun-da-re, though at the moment it automatically gets parsed as I-o-cun-da-re....maybe not the best example!

Reply:
I'll pick a different example, thanks!